src/search_with_index.py

```diff
@@ -18,8 +19,9 @@ def load_index_and_search(query, sentence_model,index_folder,max_index_memory_us
     model = config['DEFAULT']['model']
     index_folder = config['DEFAULT']['index_folder']
     max_index_memory_usage = config['DEFAULT'].get('max_index_memory_usage', '10MB')
-
     
     results = load_index_and_search(query, model, index_folder, max_index_memory_usage)
 
-    print(results)
\ No newline at end of file
+    ce = CrossencoderSearch(query,results,run_cross_encoding=True)
```

Comment:
We shouldnt remove the config flag. We want to be able to toggle it on and off in the pipeline from the config

And run_cross_encoding=True will lead to an error since we removed it from the src/CrossEncoderSearch.py code